.github/workflows/ci.yaml

```diff
@@ -20,181 +20,147 @@ on:
 jobs:
   schemachange_synapse_data_warehouse_dev:
     runs-on: ubuntu-22.04
-    if: github.ref == 'refs/heads/dev'
+    if: github.ref_name == 'dev'
     environment: dev
-    env:
-      SNOWFLAKE_PASSWORD: ${{ secrets.SNOWSQL_PWD }}
-      SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWSQL_ACCOUNT }}
-      SNOWFLAKE_USER: ${{ secrets.SNOWSQL_USER }}
-      SNOWFLAKE_SYNAPSE_DATA_WAREHOUSE_DATABASE: ${{ vars.SNOWFLAKE_SYNAPSE_DATA_WAREHOUSE_DATABASE }}
-      SNOWFLAKE_WAREHOUSE: ${{ secrets.SNOWSQL_WAREHOUSE }}
-      SNOWFLAKE_SYNAPSE_STAGE_STORAGE_INTEGRATION: ${{ vars.SNOWFLAKE_SYNAPSE_STAGE_STORAGE_INTEGRATION }}
-      SNOWFLAKE_SYNAPSE_STAGE_URL: ${{ vars.SNOWFLAKE_SYNAPSE_STAGE_URL }}
-      STACK: ${{ vars.STACK }}
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
-        with:
-          python-version: '3.10'
+      - name: Checkout repository
+        uses: actions/checkout@v4
 
-      - name: install-py-dependencies
-        shell: bash
-        run: |
-          pip install schemachange==3.6.1
-          pip install numpy==1.26.4
-          pip install pandas==1.5.3
+      - name: Configure Snowflake
+        uses: ./.github/actions/configure-snowflake-cli
+        with:
+          PRIVATE_KEY_PASSPHRASE: ${{ secrets.ADMIN_SERVICE_PASS_PHRASE }}
+          PRIVATE_KEY: ${{ secrets.ADMIN_SERVICE_PRIVATE_KEY }}
+          ACCOUNT: ${{ secrets.SNOWSQL_ACCOUNT }}
```

Comment:
To be sure: Is this `SNOWSQL_ACCOUNT` the new `ADMIN_SERVICE` account or Tom's account still?

Reply:
An _account_ in Snowflake means the Snowflake account. ADMIN_SERVICE and Tom's "account" are actually users. So this secret contains our Snowflake account identifier. I'm not sure why Tom labeled `SNOWSQL_ACCOUNT` with the client he was using to connect to the account.

Reply:
Got it!